### ISSUE TITLE
Accuracy drop using pretrained NIN model. 

### ISSUE THREAD

**Issue by analog75** [jiecaoyu/XNOR-Net-PyTorch#77]
Hi. Thank you for your uploading. I have downloaded NIN cifar10 pretrained model and load it. Then, I run the inference evaluation. At this time. the accuracy is just only 59.36%. In this evaluation, there is no modification and the model and data are used according to Readme. 
But, when I run the training using pretrained model, the accuracy is very close to the best accuracy. I think that there seems to be any dependency in train and test in your code. To solve this problem, is there any trick or solution? Thanks. 

![image](https://user-images.githubusercontent.com/55831405/65673429-c280bd80-e085-11e9-8441-3fbc412715b6.png)

**Comment by Lanweichao** [comment]
@analog75 Hello, have you solved your problem? I also obtained a lower accuracy using NIN. Thank you

**Comment by analog75** [comment]
Hi. I solve this. This code adopts random, which should be modified int pseudo-random.

 

In addition, batch normalization can make your accuracy lowered, which is very natural.

**Comment by ngocqn** [comment]
> Hi. Thank you for your uploading. I have downloaded NIN cifar10 pretrained model and load it. Then, I run the inference evaluation. At this time. the accuracy is just only 59.36%. In this evaluation, there is no modification and the model and data are used according to Readme.
> But, when I run the training using pretrained model, the accuracy is very close to the best accuracy. I think that there seems to be any dependency in train and test in your code. To solve this problem, is there any trick or solution? Thanks.
> 
> ![image](https://user-images.githubusercontent.com/55831405/65673429-c280bd80-e085-11e9-8441-3fbc412715b6.png)

I had the same issue with you.
Test set: Average loss: 9.3183, Accuracy: 1003/10000 (10.03%)
Best Accuracy: 86.28%

How to solve this issue?
